app/views/other_change_validation_requests/_form.html.erb

```diff
@@ -39,6 +39,7 @@
                                label: { text: 'Explain to the applicant how the application can be made valid.', size: 's', class: 'govuk-label govuk-label--s'},
                                hint: { text: 'Add all information that they will need to complete this action.' },
                                rows: 5 %>
+
```

Comment:
Can you please revert this?

Reply:
I quite like having a space here to separate the sections. Do you feel strongly about not having this?

Reply:
I just noticed that the first line of this file is blank . Why 🤦‍♂️